backend-rust/src/migrations/m0003.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE bakers
+-- Fraction of transaction rewards rewarded at payday to this baker pool.
+-- Stored as a fraction of an amount with a precision of `1/100_000`.
+ADD COLUMN payday_transaction_commission BIGINT,
+-- Fraction of baking rewards rewarded at payday to this baker pool.
+-- Stored as a fraction of an amount with a precision of `1/100_000`.
+ADD COLUMN payday_baking_commission BIGINT,
+-- Fraction of finalization rewards rewarded at payday to this baker pool.
+-- Stored as a fraction of an amount with a precision of `1/100_000`.
+ADD COLUMN payday_finalization_commission BIGINT;
```

Comment:
If you introduce a separate table for just this information, for the payday we can then empty this new table and insert every `payday_*_commission` for the new payday.